### PR TITLE
Update Polish (pl) localization of Firefox for iOS 170224

### PR DESCRIPTION
**Pull request by piotrdrag**
A few fixes. Please don’t push without @splewako’s approval.

### CODE REVIEW

File: pl/firefox-ios.xliff [hunk 1]
```diff
@@ -2863,7 +2863,7 @@
   <body>
    <trans-unit id="Bookmarks Menu">
     <source>Bookmarks Menu</source>
-    <target>Zakładki</target>
+    <target>Menu Zakładki</target>
```
Comment (splewako) [inline pl/firefox-ios.xliff]: No "Menu" here please.

Comment (piotrdrag) [inline pl/firefox-ios.xliff]: It says to match bookmarks.folder.menu.label on Android, and there it’s one of the folders under “Zakładki”: Pasek zakładek, Menu Zakładki and Pozostałe zakładki.

So using just Zakładki instead would make it a “Zakładki” submenu under the “Zakładki” menu, which is highly confusing to use.

Comment (splewako) [inline pl/firefox-ios.xliff]: > It says to match bookmarks.folder.menu.label on Android

The note says that indeed, don't know why.

> and there it’s one of the folders under “Zakładki”: Pasek zakładek, Menu Zakładki and Pozostałe zakładki.
>
> So using just Zakładki instead would make it a “Zakładki” submenu under the “Zakładki” menu, which is highly confusing to use.

Is it? Aren't you confusing Android UI with how it actually looks on iOS after reading misleading comment?

I don't have Android device (or time to investigate the code) to be sure but I see something like:

- Zakładki z komputera
  - **Zakładki**
    - No recently added bookmarks from desktop menu
    - No Pasek zakładek special folder from desktop menu
    - No Pozostałe zakładki special folder from desktop menu
    - No custom special folders added on the desktop to the menu
    - Feed bookmark folders as bookmarks
    - Regular desktop bookmarks
    - Regular desktop folders
  - Pasek zakładek
  - Zakładki nieposortowane (changed recently to "Pozostałe zakładki" on desktop)
    - All bookmarks without folder matching from desktop
  - Zakładki z telefonu
    - Bookmark from Firefox for iOS nightly?
- Local bookmark 1
- Local bookmark 2
- Local bookmark 3
- Local bookmark 4

Comment (piotrdrag) [inline pl/firefox-ios.xliff]: For the record, this is how it looks on Android:

screenshot_20170228-184845

I’ve removed “Menu” from the string in firefoxios.

**Pull request merged**